design/oss_suite_release.md

```diff
@@ -0,0 +1,313 @@
+|_so that_|OSS users can benefit from new features my team is adding to our component.|
+
+|||
+|---|---|
+|_As a_|maintainer of a Conjur OSS suite component|
+|_I want_|to know what I'm responsible for testing and what will be tested in the OSS suite|
+|_so that_|it's clear what tests I need to include and maintain in my component repository, and which tests belong in the central OSS suite release repo.|
+
+|||
+|---|---|
+|_As a_|maintainer of a Conjur OSS suite component|
+|_I want_|to know what standards I need to maintain in my repository|
+|_so that_|my CHANGELOG entries, releases, READMEs, etc are compatible with the OSS suite release.|
+
+### Conjur OSS Contributors
+
+|||
+|---|---|
+|_As a_|Conjur Open Source contributor|
+|_I want_|to know what components there are|
+|_so that_|I can find components that I can contribute to.|
+
+|||
+|---|---|
+|_As a_|Conjur Open Source contributor|
+|_I want_|to know what is included in the Conjur OSS suite|
+|_so that_|if there are tools or platforms that are not supported, I can make a difference by building something to fill the gap.|
+
+### Conjur OSS Suite Release Managers
+
+|||
+|---|---|
+|_As a_|Conjur release manager|
+|_I want_|o know what changes have been made to components since the last suite release|
+|_so that_|I can decide if enough changes have been made to merit a new release.|
+
+|||
+|---|---|
+|_As a_|Conjur release manager|
+|_I want_|to know how to create a new release, including any manual steps that are required|
+|_so that_|I can successfully create a new release without missing any steps.|
+
+|||
+|---|---|
+|_As a_|Conjur release manager|
+|_I want_|to know the results of automated tests|
+|_so that_|I can be sure all included components work together as expecteed at the pinned versions.|
+
+
+### Conjur Core Downstream Developer
+|||
+|---|---|
+|_As a_|developer on a downstream project that uses Conjur as its core|
+|_I want_|to know when there are new Conjur OSS suite releases|
+|_so that_|I can review the changes and update my project to use the latest OSS suite release version of Conjur.|
+
+|||
+|---|---|
+|_As a_|developer on a downstream project that uses Conjur as its core|
+|_I want_|a machine readable summary of what is included in each OSS suite release|
+|_so that_|I can build automation around the OSS suite releases as needed.|
+
+## High-Level Technical Design
+
+The [OSS Suite Release Repo](https://github.com/cyberark/conjur-oss-suite-release)
+will include:
+- A user-facing README with a high level overview of how Conjur OSS end users
+  should leverage the repo, with links to appropriate pages in the official documentation
+- A VERSION file that includes the current version of the OSS suite release
+- A machine-readable file with the pinned versions included in the current release
+- Version tags for each OSS suite release version that correspond to GitHub releases
```

Comment:
Is an OSS suite release a new thing starting with this, or will version tags be put in for past releases for customers who are still using older components, perhaps?

Reply:
it's new! we'll be deciding what version to call it and which component versions to include in the course of implementation. we hope that current users will find the suite release a helpful way to determine which component versions to upgrade to, if they're considering an upgrade. definitely something I'd be glad to talk about on [discourse](https://discuss.cyberarkcommons.org) with end users anytime, too.